test/__snapshots__/PublishCommand.js.snap

```diff
@@ -355,6 +355,37 @@ Array [
 ]
 `;
 
+exports[`[normal --skip-git-push --skip-npm] bumps package versions, commits, and tags 1`] = `
+Object {
+  "packages/package-1": "1.0.1",
+  "packages/package-2": "1.0.1",
+  "packages/package-3": "1.0.1",
+  "packages/package-4": "1.0.1",
+  "packages/package-5": "1.0.1",
+}
+`;
+
+exports[`[normal --skip-git-push] npm publish --tag 1`] = `
```

Comment:
When adding a `toMatchSnapshot()` in a test, it is appropriate to commit the update to the snapshot file.

However, in this case, as I mentioned in the test above, I don't think this snapshot is necessary (it's almost verbatim from another snapshot).